AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -193,7 +193,6 @@ freundin.de,utopia.de,mein-schoener-garten.de,eveningnews24.co.uk,rollingstone.d
 !
 ! NOTE: Regular rules
 !
-freddy.com##.amgdprcookie-bar-template
```

Comment:
Do not remove this rule, please revert it.

Reply:
ô.o I haven't touch that rule at all